src/status_im2/navigation/screens.cljs

```diff
@@ -243,18 +244,25 @@
      :component emoji-picker/view}
 
     {:name      :wallet-accounts
+     :options   {:insets {:top? true}}
      :component wallet-accounts/view}
 
     {:name      :wallet-edit-account
      :component wallet-edit-account/view}
 
     {:name      :wallet-address-watch
+     :options   {:insets {:top? true}}
      :component wallet-address-watch/view}
 
+    {:name      :wallet-bridge
+     :options   {:insets {:top? true}}
```

Comment:
We need to add another option (for iOS) to ensure the page is filled user's device screen size
```clojure
 :modalPresentationStyle :overCurrentContext
```

<img width="300" src="https://github.com/status-im/status-mobile/assets/19339952/f9c9bbb7-ddda-412f-9a8d-19c05fb9c9f4" />

Reply:
I am wondering if this should be a modal or opened with `navigate-to`

@briansztamfater I see you made the `select address / send` screen as modal. Do we have this documented somewhere? Or should it be opened with `navigate-to` ?